File: sdkwork-app-sdk-typescript/src/types/share-track-form.ts
/** 分享追踪请求 */
export interface ShareTrackForm {
  /** 分享ID */
  shareId?: string;
  /** 追踪类型 */
  trackType?: string;
  /** 用户代理 */
  userAgent?: string;
  /** IP地址 */
  ipAddress?: string;
  /** 设备信息 */
  deviceInfo?: string;
}

File: sdkwork-app-sdk-go/types/share_track_form.go
package types

// 分享追踪请求
type ShareTrackForm struct {
	ShareId string `json:"shareId"`
	TrackType string `json:"trackType"`
	UserAgent string `json:"userAgent"`
	IpAddress string `json:"ipAddress"`
	DeviceInfo string `json:"deviceInfo"`
}
